examples/ota-provider-app/linux/main.cpp

```diff
@@ -54,12 +54,18 @@ constexpr uint16_t kOptionFilepath             = 'f';
 constexpr uint16_t kOptionOtaImageList         = 'o';
 constexpr uint16_t kOptionQueryImageBehavior   = 'q';
 constexpr uint16_t kOptionDelayedActionTimeSec = 'd';
+constexpr uint16_t kOptionDiscriminator        = 'D';
```

Comment:
nit: folks who use this app most likely also uses the Linux ota-requestor-app which uses `-d` as the discriminator command line option. Maybe to keep the two consistent, we should keep this `-d` and perhaps delayed action time can be `-t` or something?